ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -246,7 +277,26 @@ def array_dtypes(
 
     Returns
     -------
-    A strategy that draws a list.
+        A strategy that draws a list of data types.
+
+    Examples
+    --------
+    >>> array_dtypes(
+    ...     available_dtypes=get_dtypes("numeric"),
+    ...     shared_dtype=True,
+    ... )
+    ['float64']
+    ['int8', 'int8']
```

Comment:
Let's only provide one output of the single function call, so not to cause confusion.